src/main/target/FRSKYF3/target.h

```diff
@@ -0,0 +1,195 @@
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "FRF3"  
+#define TARGET_CONFIG
+
+#define CONFIG_FASTLOOP_PREFERRED_ACC ACC_NONE
+
+#define LED0                    PB3
+
+#define BEEPER                  PC15
+#define BEEPER_INVERTED
+
+#define USE_EXTI
+#define MPU_INT_EXTI PC13
+#define USE_MPU_DATA_READY_SIGNAL
+#define EXTI15_10_CALLBACK_HANDLER_COUNT 1 // MPU_INT, SDCardDetect
+
+#define MPU_ADDRESS             0x69
+
+
+//#define MYMPU6000
+
+#ifdef MYMPU6000
+
+#define MPU6000_SPI_INSTANCE    SPI2
+#define MPU6000_CS_PIN          PB12
+
+#define GYRO
+#define USE_GYRO_SPI_MPU6000
+#define GYRO_MPU6000_ALIGN      CW270_DEG
+//CW90_DEG
+
+#define ACC
+#define USE_ACC_SPI_MPU6000
+#define ACC_MPU6000_ALIGN       CW270_DEG
+//CW90_DEG
+#else
+#define GYRO
+
+#define USE_GYRO_MPU6050
+#define GYRO_MPU6050_ALIGN      CW270_DEG
+//CW270_DEG
+//CW90_DEG
+#define ACC
+
+#define USE_ACC_MPU6050
+#define ACC_MPU6050_ALIGN       CW270_DEG
+//CW270_DEG
+//CW90_DEG
+
+
+#endif
+
+
+#define USE_VCP
+#define USE_UART1
+#define USE_UART2
+#define USE_UART3
+#define USE_SOFTSERIAL1
+#define USE_SOFTSERIAL2
+
+#define SERIAL_PORT_COUNT       6
+
+#define UART1_TX_PIN            PA9
+#define UART1_RX_PIN            PA10
+
+#define UART2_TX_PIN            PA14 
+#define UART2_RX_PIN            PA15
+
+#define UART3_TX_PIN            PB10 
+#define UART3_RX_PIN            PB11 
+
+#undef USE_I2C
+#define USE_I2C
+#define USE_I2C_DEVICE_1
+#define I2C_DEVICE              (I2CDEV_1)
+#define I2C1_SCL                PB6
+#define I2C1_SDA                PB7
+
+
+#define USE_ESCSERIAL
+#define ESCSERIAL_TIMER_TX_HARDWARE 0 
+
+#define USE_SPI
+
+#define OSD
+
+// include the max7456 driver
+#define USE_MAX7456
+#define MAX7456_SPI_INSTANCE    SPI2
+#define MAX7456_SPI_CS_PIN      PB4
+#define MAX7456_SPI_CLK         (SPI_CLOCK_STANDARD*2)
+#define MAX7456_RESTORE_CLK     (SPI_CLOCK_FAST)
+
+
+#define USE_SPI
+#define USE_SPI_DEVICE_2 
+#define USE_SPI_DEVICE_1 
+
+
+#define SPI2_NSS_PIN            PB12
+#define SPI2_SCK_PIN            PB13
+#define SPI2_MISO_PIN           PB14
+#define SPI2_MOSI_PIN           PB15
+
+
+#define SPI1_NSS_PIN            PC14
+#define SPI1_SCK_PIN            PA5
+#define SPI1_MISO_PIN           PA6
+#define SPI1_MOSI_PIN           PA7
+
+
+#define USE_SDCARD
+#define USE_SDCARD_SPI1
+
+#define SDCARD_DETECT_INVERTED
+
+#define SDCARD_DETECT_PIN                   PB5
+#define SDCARD_SPI_INSTANCE                 SPI1
+#define SDCARD_SPI_CS_PIN                   SPI1_NSS_PIN
+
+
+#define SDCARD_SPI_INITIALIZATION_CLOCK_DIVIDER 256
+
+#define SDCARD_SPI_FULL_SPEED_CLOCK_DIVIDER     4  
+
+#define USE_ESC_SENSOR
+
+
+
```

Comment:
There is a lot of empty lines, and unadjusted indentation in all of the files. Please clean up.